### DISCUSSION TITLE
Facing issue after installing apk

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#651] [General Discussion]
@swayangjit I have generated the apk of sdk 33.
APK is installed and when i open it i am seeing the blank white screen and this issue in console 

![image](https://github.com/Sunbird-Ed/Community/assets/72926048/47d10eb8-ab9b-464f-bd68-1bc9f3b91acf)

**Comment by swayangjit** [comment]
@siddhishinde0723 sbutility plugin is not installed . Please install it and check.

**Reply by siddhishinde0723** [reply]
@swayangjit Its installed.

**Reply by swayangjit** [reply]
Some how it's not installed.Just try to install it again.

**Reply by siddhishinde0723** [reply]
@swayangjit Thanks this is solved
